ramls/retrieveEntitiesDto.json

```
@@ -0,0 +1,25 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "DTO for fetching Holding and Instances by POST request",
```

```suggestion
  "description": "DTO for fetching records by POST request",
```
I think it could use for other entities too in the future.